packages/commonwealth/client/scripts/views/components/Comments/CommentEditor/CommentEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { CWText } from '../../component_kit/cw_text';
+import { User } from 'views/components/user/user';
+import Topic from 'client/scripts/models/Topic';
+import Account from 'client/scripts/models/Account';
+import app from 'state';
+import clsx from 'clsx';
+import { ReactQuillEditor } from '../../react_quill_editor';
+import { CWButton } from '../../component_kit/new_designs/cw_button';
+import { CWValidationText } from '../../component_kit/cw_validation_text';
+import BN from 'bn.js';
+import { weiToTokens, getDecimals } from 'helpers';
+import { ContentType } from 'types';
+import type { DeltaStatic } from 'quill';
+
+type CommentEditorProps = {
+  parentType: ContentType;
+  canComment: boolean;
+  handleSubmitComment: () => void;
+  errorMsg: string;
+  contentDelta: DeltaStatic;
+  setContentDelta: React.Dispatch<React.SetStateAction<DeltaStatic>>;
+  tokenPostingThreshold: BN;
+  activeTopic: Topic;
+  userBalance: BN;
+  disabled: boolean;
+  cancel: (e: any) => void;
+  isAdmin: boolean;
+  author: Account;
+  editorValue: string;
+  shouldFocus: boolean;
+};
+
+export const CommentEditor = ({
+  parentType,
+  canComment,
+  handleSubmitComment,
+  errorMsg,
+  contentDelta,
+  setContentDelta,
+  tokenPostingThreshold,
+  activeTopic,
+  userBalance,
+  disabled,
+  cancel,
+  isAdmin,
+  author,
+  editorValue,
+  shouldFocus,
+}: CommentEditorProps ) => {
+
+  const decimals = getDecimals(app.chain);
+
+  return (
+    <div className="CreateComment">
```

Comment:
this is great that you decided to move comment editor to separate file 💪 but if we do this, we should remember about the styles as well and general rules we have for creating files/componnets/namings etc. So in this case:
- main class name should be changed from `CreateComment` to `CommentEditor`
- new scss file `CommentEditor.scss` should be created that would handle only styles for this particular component

Those steps are important to have files and codebase tidy up.

Reply:
Done!